cli/src/test/resources/ide-projects/update/_ide/urls/java/java/17.0.6/linux_x64.urls

```diff
@@ -0,0 +1 @@
+http://localhost:1111/installTest/linux
```

Comment:
seems all like copy&paste stuff that is not needed anywhere in your tests that work entirely without wiremock.
So please omit the entire `urls` folder here if it is not needed. We will fully lose control of maintenance if we add tons of  clutter and nobody knows what is actually needed and relevant and why something is there...

Reply:
done